vscode/src/webview/help.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function HelpPage() {
+  return (
+    <div style="line-height: 1.5">
+      <h1>Azure Quantum Development Kit</h1>
+      <h2>Overview</h2>
+      <p>
+        Welcome to the Azure Quantum Development Kit! Or the "QDK", as we like
```

Comment:
Obviously this help content needs to be fleshed out, but want to get eyes on the rest of the PR ASAP.